include/dfrn.php

```diff
@@ -424,11 +426,17 @@ public static function relocate($owner, $uid) {
 
 		$rp = q("SELECT `resource-id` , `scale`, type FROM `photo`
 				WHERE `profile` = 1 AND `uid` = %d ORDER BY scale;", $uid);
+
```

Comment:
It would be better to just don't export the profile pictures when that query had been empty. This is better than committing suicide.

Concerning ```dbm::is_result```: It will return ```true``` whenever a query was successful or contains at least one line. This change was needed for the transition towards the new database functions.

Concerning I personally don't like a too high abstraction layer. There will always be the need for some low level calls.